internal/bminventory/inventory.go

```diff
@@ -937,9 +940,11 @@ func (b *bareMetalInventory) UpdateCluster(ctx context.Context, params installer
 	}
 
 	if sshPublicKey := swag.StringValue(params.ClusterUpdateParams.SSHPublicKey); sshPublicKey != "" {
+		sshPublicKey := strings.TrimSpace(sshPublicKey)
```

Comment:
Maybe you can validate that there are no spaces in the middle.

Reply:
i can add it to validation, i think will do it in another PR

Reply:
Please open a ticket for it

Reply:
tested it. Validation will fail if there is \n in the middle

Reply:
@filanov this cases are already covered (checked)